### ISSUE TITLE
How to use webhook -header to enable XSS protection

### ISSUE THREAD

**Issue by SJebasingh** [adnanh/webhook#553]
How to use webhook -header to enable XSS protection

**Comment by SJebasingh** [comment]
I tried to use as below and not working.

Webhook.exe - port 90 - header X-Frame-Options=SAMEORIGIN

When I tried to get headers using curl I'm not able to get those headers. I want to fix vulnerabilities.

Curl -lkL --verbose http://localhost:90

**Comment by SJebasingh** [comment]
Yes. It works however security scan is run on port and ip as below and not on with added hooks.

http://localhost:90

**Comment by moorereason** [comment]
BTW, a fix for this issue has been merged into the development branch.
